packages/manager/modules/backup-agent/src/components/CommonCells/ResourceStatusCell/__tests__/ResourceStatusCell.component.test.tsx

```diff
@@ -1,35 +1,31 @@
-import React from 'react';
-
 import { render, screen } from '@testing-library/react';
 import { describe, expect, it, vi } from 'vitest';
 
 import { mockVaults } from '@/mocks/vaults/vaults';
-import { ResourceStatusCell } from '../ResourceStatusCell.components';
 import { ResourceStatus } from '@/types/Resource.type';
 
+import { ResourceStatusCell } from '../ResourceStatusCell.component';
+
 vi.mock('react-i18next', () => ({
   useTranslation: () => ({ t: (key: string) => key }),
 }));
 
 vi.mock('@/components/ResourceStatusBadge/ResourceStatusBadge.components', () => ({
```

Comment:
```suggestion
vi.mock('@/components/ResourceStatusBadge/ResourceStatusBadge.component', () => ({
```

This for that this not work now ;)